Makefile

```diff
@@ -45,18 +45,18 @@ download:
 	mkdir -p intermediate_source/data
 
 	# transfer learning tutorial data
-	wget -N https://download.pytorch.org/tutorial/hymenoptera_data.zip -P $(DATADIR)
+	wget -N http://download.pytorch.org/tutorial/hymenoptera_data.zip -P $(DATADIR)
```

Comment:
Why the switch from `https` -> `http`?

I'd much prefer it to use `https` over regular `http`?

Reply:
> Why the switch from `https` -> `http`?
> 
> I'd much prefer it to use `https` over regular `http`?

fixed